roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml

```diff
@@ -10,6 +10,11 @@
 
 - name: Make sure heketi-client is installed
   package: name=heketi-client state=present
+  when: not openshift.common.is_atomic | bool
+
+- name: Verify heketi-cli is installed
+  shell: "command -v heketi-cli >/dev/null 2>&1 || { echo >&2 'ERROR: heketi-cli must be installed.'; exit 1; }"
```

Comment:
I'd recommend something like this (totally untested 😆)

```yaml
- name: Check if heketi-cli is installed
  shell:  "which heketi-cli"
  register: l_command_exists
  changed_when: false

# Verify the command exists
- assert:
    that:  l_command_exists.rc == 0
  msg: "heketi-cli must be installed on the host"
```
**However** if this ``heketi-cli`` is not part of Atomic Host and there isn't a current want to place it on there it may be easier to do an assert at the start of this file that stops execution if ``is_atomic`` with a note that this needs to be expanded to work on Atomic Host in the near future.